tests/unit/test_network_connections.py

```diff
@@ -317,13 +317,19 @@ def do_connections_validate_ifcfg(self, input_connections, **kwargs):
             content_current = kwargs.get("initscripts_content_current", None)
             if content_current:
                 content_current = content_current[idx]
-            c = IfcfgUtil.ifcfg_create(
-                connections, idx, content_current=content_current
+            warnings = []
+            config = IfcfgUtil.ifcfg_create(
+                connections,
+                idx,
+                content_current=content_current,
+                warn_fcn=lambda msg: warnings.append(msg),
```

Comment:
Nice finding from pylint:

```suggestion
                warn_fcn=warnings.append,
```